components/TicketsSection.js

```diff
@@ -5,7 +5,7 @@ export default function TicketsSection () {
     <div className='tickets'>
       <SectionTitle>Tickets</SectionTitle>
       <TicketsTable />
-      <Button href='https://www.eventpop.me/e/1809-react-bangkok-2-0-0'>Get Tickets</Button>
+      <Button href='https://www.eventpop.me/e/1809-react-bangkok-2-0-0' date='25 May 2017'>Get Tickets</Button>
```

Comment:
The button should not know about the date. At most, it should only know which color it show show, e.g. `color='accent'`, `color='highlight'`.